test/controllers/api/users_controller_test.rb

```diff
@@ -419,13 +419,15 @@ def test_index
       check_json_details(js["users"][1], user3, false, false)
 
       get api_users_path, :params => { :users => create(:user, :suspended).id }
-      assert_response :not_found
+      assert_response :success
```

Comment:
You should add the same assertion here that you use later to check that no users are returned.